### ISSUE TITLE
V2.4.12 --ec with pipe from stdin generates an error.

### ISSUE THREAD

**Issue by bgulko** [bedops/bedops#144]
> bedops --ec -m -
May use bedops --help for more help.

Error: Error: stat() failed on: stdin

With 2.4.11, this worked fine. Removing --ec in 2.4.12 returns bedops behavior to normal.
-- Brad

**Comment by alexpreynolds** [comment]
Seems there may be an issue:

```
$ bedops --version
bedops
  citation: http://bioinformatics.oxfordjournals.org/content/28/14/1919.abstract
  version:  2.4.12
  authors:  Shane Neph & Scott Kuehn

$ cat foo.bed | bedops --ec -m - | head
May use bedops --help for more help.

Error: Error: stat() failed on: stdin

$ cat foo.bed | bedops -m - | head
A01	158188	158536
A01	160347	160578
A01	888838	889441
...
```

**Comment by sjneph** [comment]
Thanks for the report.  The issue is fixed in commit acc3d75d8cfb0b9f2426d9a6e219627552a95e6b of the v2p4p13 branch.  I suggest that we release that branch officially.

**Comment by alexpreynolds** [comment]
v2.4.13 is now available here: https://github.com/bedops/bedops/releases/tag/v2.4.13
